### DISCUSSION TITLE
Ajuda com o Visual(HTML) da Página

### DISCUSSION THREAD

**Discussion by iramaceber** [LiveHelperChat/livehelperchat#2013] [Q&A]
Bom Dia, eu estou com um problema depois da instalação. Já pesquisei várias vezes sobre esse erro mas não encontro em lugar nenhum.
![error](https://github.com/LiveHelperChat/livehelperchat/assets/44277796/a729102d-02fd-425b-802c-cc15d6f5a98a)

**Comment by remdex** [comment]
That's your issue... https://doc.livehelperchat.com/docs/debug
